src/main/java/com/mate/model/Ticket.java

```diff
@@ -0,0 +1,52 @@
+package com.mate.model;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+
+@Entity
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
Okey, you need those joins. Let's check which requests would be more convenient(and how many we will have). 
1) you are using here Lazy and making all of those joins;
2) you are using hefre EAGER and joining only tickets;
You can watch the SQL query which hibernate does using property show-sql

Reply:
You can paste screenshots here or in your chat